netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -79,14 +79,18 @@
    * noticing changes to DNS.
    */
   public static NettyChannelBuilder forAddress(SocketAddress serverAddress) {
```

Comment:
Yes. This method would go away completely if it weren't needed to use specialized channels.